schemas/metaschema/DataStructure.schema

```diff
@@ -0,0 +1,3 @@
+{
+  Symbol.for('AllowNotDefinedFields'): false
```

Comment:
That's an invalid syntax, you have to enclose the computed property name into the brackets:
```suggestion
  [Symbol.for('AllowNotDefinedFields')]: false
```